week1/homework/src/server.js

```diff
@@ -1,20 +1,47 @@
 'use strict';
 
 const http = require('http');
-
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
-
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    response.setHeader('Content-Type', 'application/json');
+    if (request.url === '/state') {
```

Comment:
`else if` works perfectly well, but for brevity and readability, we might want to prefer `switch` cases where we have one variable and many possible values.

Reply:
ok , I'll change it to switch :)